subseas_flex_fcst/layout.py

```diff
@@ -29,11 +29,18 @@
 def app_layout():
 
     # Initialization
+    if CONFIG["leads"] != "none":
+        use_leads = list(CONFIG["leads"])[0]
+        use_targets = None
+    else:
+        use_leads = None
+        use_targets = CONFIG["targets"][1]
```

Comment:
When there are two mutually-exclusive options, I like to raise an exception if both are specified, rather than silently ignoring one of them. It makes debugging easier.

Reply:
Let me know if that is not the way to go